### ISSUE TITLE
Dinner

### ISSUE THREAD

**Issue by Rob1Paul** [ReadWithAudrey/readwithaudrey#161]
Hi @matthall8 @BenNewman90 @DenisKent @sbinlondon I would like to take you all out for dinner to say thank you for all your hard work. Are you around this coming Monday evening?

**Comment by sbinlondon** [comment]
Hi @Rob1Paul that sounds lovely. We've got a coding event going on Monday, but on Wednesday we're all free. We'll actually be in the Old St area for a coding workshop during the day and finishing up around 5 pm I think, so perhaps we can all meet there for dinner? I used to work around there and know tons of great places to eat. If that works for everyone we can suggest what people fancy eating and make a choice. I heard the guys love falafel after Nazareth...
